### PR TITLE
Added env variables for DB credentials and fixed some formatting

### PR DESCRIPTION
**Pull request by Ty-Won**
Changed the DB credentials to use env variables, I will post the .env file on slack later so that everyone can use this.

### CODE REVIEW

File: backend/routes/users.js [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+router.post("/completedCourses/comparison", function(req, res) {
+  let completedCourses = req.body;
+  curriculum
+    .courseComparison(completedCourses)
+    .then(function(remainingCourses) {
+      res.send(remainingCourses);
+    })
+    .catch(function(err) {
+		console.error(err);
+		res.send(/*TODO ERROR CODE*/);
```
Comment (si-x-li) [inline backend/routes/users.js]: TODO code left here

File: backend/logic/users/users.js [hunk 2]
```diff
@@ -1,94 +1,115 @@
-const mysql = require('../../sql/connection');
```
Comment (si-x-li) [inline backend/logic/users/users.js]: Possible merge conflicts with #28. Will need to be resolved

File: backend/routes/users.js [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
-	let email = req.query.email;
-	let id = req.query.student_id;
-	users.insertStudentUser(username, password, email, id)
-	.then(function(val) {
-		console.log(val);
-		res.send(val);
+router.post("/addStudentUser", function(req, res, next) {
+  let username = req.body.username;
+	let password = req.body.password;
+	let email = req.body.email;
```
Comment (si-x-li) [inline backend/routes/users.js]: Formatting is broken

File: backend/logic/users/users.js (hunk 2 above)
Comment (Ty-Won) [reply backend/logic/users/users.js]: is it because of the single quotes? 0-0

Comment (si-x-li) [reply backend/logic/users/users.js]: users.js is formatted in #28 and this

Comment (KevenLYF) [reply backend/logic/users/users.js]: we can merge this first, then I'll fix my commit #28, then we merge #28

File: backend/routes/users.js (hunk 1 above)
Comment (KevenLYF) [inline backend/routes/users.js]: we can send 500 it it's a server error (Backend error)

File: backend/routes/users.js (hunk 3 above)
Comment (KevenLYF) [reply backend/routes/users.js]: once we fixed this formatting we can merge this

**Comment by Ty-Won** [comment]
@si-x-li @KevenLYF I chose to force update since the changes were very small and minor, no need to create extra commit history with no purpose

**Pull request merged**